src/build/HHAPIDocBuildStep.php

```diff
@@ -47,10 +47,12 @@ public function buildAll(): void {
       $exts,
     );
     Log::i("\nParsing builtins");
-    list($runtime_defs, $hhi_defs) = \HH\Asio\join(Tuple\from_async(
-      self::parseAsync($runtime_sources),
-      self::parseAsync($hhi_sources),
-    ));
+    list($runtime_defs, $hhi_defs) = \HH\Asio\join(async {
+      return tuple(
+        await self::parseAsync($runtime_sources),
+        await self::parseAsync($hhi_sources),
+      );
```

Comment:
This makes it not concurrent anymore, doesn't it? Is wrapping this in `concurrent` legal? If not, we may need something like

```
concurrent {
  $a = ...
  $b = ...
}
return tuple($a, $b);
```

Reply:
multiple awaits in the same expression are concurrent